src/components/PersonCardList/index.js

```diff
@@ -1,32 +1,36 @@
-import React, { Component } from "react";
+import React from "react";
+
 import { withRouter } from "react-router";
 
 import ArrowRight from "../../icons/ArrowRight";
 
-class PersonCardList extends Component {
-  render() {
-    const { data, history, startPath } = this.props;
-    let startPathCheck = "";
-    if (startPath !== undefined) {
-      startPathCheck = startPath;
-    }
-    return data.map((item, index) => (
-      <div key={index} className="client__card">
-        <img src={`${startPathCheck}${item.mainPhoto}`} alt="client image" />
-        <h3 className="client__name">
-          {item.firstName} {item.lastName}
-        </h3>
-        <h4 className="client__role">{item.type}</h4>
-        <a
-          onClick={() => history.push(`/profile/${item.id}`)}
-          className="client__link"
-          href={item.link}
-        >
-          <ArrowRight />
-        </a>
-      </div>
-    ));
+const PersonCardList = ({ data, history, startPath }) => {
+  let startPathCheck = "";
+  if (startPath !== undefined) {
```

Comment:
??? what is it 
 `if (startPath !== undefined) {`

 -> 

`if (startPath) {`